Source/FORMData.m

```diff
@@ -875,11 +875,13 @@ - (NSArray *)updateTargets:(NSArray *)targets
 
 - (NSArray *)enableTargets:(NSArray *)targets
 {
+    [[NSNotificationCenter defaultCenter] postNotificationName:FORMHideTooltips object:[NSNumber numberWithBool:YES]];
```

Comment:
Do you mind using a literals here? 

`@YES` => `[NSNumber numberWithBool:YES]`

Method formatting `:` goes in newlines
```objc
[[NSNotificationCenter defaultCenter] postNotificationName:FORMHideTooltips
                                                    object:@YES];
```